### PR TITLE
Update basic MDS description

### PR DESCRIPTION
**Pull request by jan--f**

### CODE REVIEW

File: xml/admin_about.xml [hunk 1]
```diff
@@ -285,9 +285,10 @@
      <term>&mds; (MDS)</term>
      <listitem>
       <para>
-       The &mds;s store metadata for the &cephfs;. By using an MDS you can
-       execute basic file system commands such as <command>ls</command> without
-       overloading the cluster.
+       &cephfs; metadata is stored in its own rados pool (see
```
Comment (asettle) [inline xml/admin_about.xml:288]:
```suggestion
       &cephfs; metadata is stored in its own RADOS pool (see

File: xml/admin_about.xml [hunk 2]
```diff
@@ -285,9 +285,10 @@
...
+       <xref linkend="storage-intro-structure-pool"/>). 
+       The &mds;s act as a smart caching layer for the metadata and serializes access when needed.
+       This allows concurrent access from many clients without explicit syncronisation.
```
Comment (asettle) [inline xml/admin_about.xml:291]:
```suggestion
       This allows concurrent access from many clients without explicit synchronization.

**Review by asettle — COMMENTED** [review]
Small nits.

**Review by asettle — APPROVED** [review]

File: xml/admin_about.xml [hunk 3]
```diff
@@ -285,9 +285,10 @@
      <term>&mds; (MDS)</term>
      <listitem>
       <para>
-       The &mds;s store metadata for the &cephfs;. By using an MDS you can
-       execute basic file system commands such as <command>ls</command> without
-       overloading the cluster.
+       &cephfs; metadata is stored in its own RADOS pool (see
```
Comment (tbazant) [inline xml/admin_about.xml:288]:
```suggestion
       &cephfs; metadata is stored in its own &rados; pool (see
```

**Review by tbazant — APPROVED** [review]
lgtm

**Pull request merged**